docs/query_proxy.md

```diff
@@ -153,6 +155,25 @@ The proxy server monitors the permissions file for changes. Whenever a change is
 it passes validation, switches to the new version. Care should be taken when editing the file while the proxy server is running, because
 as soon as you save the file, the changes will be picked up (whether they are logically complete or not).
 
+#### The `allowAnything` flag
+
+If this flag is specified for a user, then that user may make any call on any supported chain, without restriction.
```

Comment:
Would it make sense to reiterate that this is only allowed in devnet/testnet here?

Reply:
Seems redundant?